### PR TITLE
Fix: Issue 2551

### LINKED ISSUE #2551: [Localized Vacuum Cleaner] On reset last position is mark as cleaned
When reseting the excercise the last point will remain.

Here is a trace of console logs to see what is happening:

- At the top is the last position of the previous iteration
- Reset
- On the first iteration the Specific js from the excercise tells the helper to draw in one position but in the helper the values received are from the last execution.

image

Here is a branch with the modified files to test: https://github.com/javizqh/RoboticsAcademy/tree/issue-2551

### PR DESCRIPTION
**Pull request by Apoorvgarg-creator**
Fix #2551

### CODE REVIEW

File: exercises/templates/exercises/vacuum_cleaner_loc_newmanager/exercise.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {% endblock %}
 
 {% block react-content %}
-    {% react_component exercise/vacuum_cleaner_newmanager/BasicVacuumCleaner %}
+    {% react_component exercise/vacuum_cleaner__loc_newmanager/BasicVacuumCleaner %}
```
Comment (pawanw17) [inline exercises/templates/exercises/vacuum_cleaner_loc_newmanager/exercise.html]: Is that a typo?

**Review by javizqh — CHANGES_REQUESTED** [review]
It still happens

https://github.com/JdeRobot/RoboticsAcademy/assets/124056014/6f756b7e-8b7e-4ffa-a47f-93a3290fb097

**Comment by Apoorvgarg-creator** [comment]
@javizqh Robot position started from initial, right ?
But there was one blue mark at the last position ?

Expectation from this video is,

That last position blue mark shouldn't be there.
Am I right ?

**Comment by javizqh** [comment]
Yes, that dot should not appear

**Comment by javizqh** [comment]
Everything else is correct, is just the dot that appears at the last position of the last execution

**Comment by Apoorvgarg-creator** [comment]
Whenever I try to run any solution I get this error.


Screenshot 2024-07-02 at 2 06 09 AM

**Comment by Apoorvgarg-creator** [comment]
@javizqh Can you please test the changes

**Review by javizqh — CHANGES_REQUESTED** [review]
Sadly it still happens. Use this code to test it, to see if for you it works OK:
``` python
import GUI
import HAL
# Enter sequential code!

angular_speed = 0.3
linear_speed = 0.2
i = 0

while True:
    HAL.setV(linear_speed)
    HAL.setW(angular_speed)
    i = i + 1
```

**Comment by Apoorvgarg-creator** [comment]
Demo Video to show the PR works.

https://github.com/JdeRobot/RoboticsAcademy/assets/57873504/2fb23383-e09b-4543-b0fe-6503af45e5f2

@javizqh

**Review by javizqh — APPROVED** [review]
Works perfectly

**Pull request merged**